core/src/main/scala/com/rallyhealth/vapors/factfilter/evaluator/package.scala

```diff
@@ -0,0 +1,30 @@
+package com.rallyhealth.vapors.factfilter
+
+import com.rallyhealth.vapors.core.interpreter
+
+package object evaluator {
+
+//  @deprecated("Use com.rallyhealth.vapors.core.interpreter.ExprInput instead.", "0.8.0")
```

Comment:
I assume that this will be removed?  (I'm going commit by commit...)

Reply:
I added the aliases to make sure the code still compiled (mostly), then I removed the aliases and found all the spots that failed to compile and updated them to use the new package paths, and then uncommented the aliases and committed the change... however, it looks like I forgot the last step on this commit.